src/Passes/requirements.txt

```diff
@@ -0,0 +1 @@
+click==8.0.1
```

Comment:
Would be nice to have some info/link telling which entity uses this file.

Reply:
This file is used to setup the environment. It is used in the [README.md](https://github.com/troelsfr/qsharp-compiler/blob/feature/llvm-pass-proposal/src/Passes/README.md) under the section `CI` to setup the environment to executing the manage command for the CI processes.

Reply:
Thank you.